src/python/grpcio_observability/grpc_observability/_open_census_exporter.py

```diff
@@ -12,23 +12,293 @@
+                options, interval=CENSUS_UPLOAD_INTERVAL_SECS
+            )
+            self.view_manager.register_exporter(metrics_exporter)
+            self._register_open_census_views()
+
+        if self.config.tracing_enabled:
+            current_tracer = execution_context.get_opencensus_tracer()
+            trace_id = current_tracer.span_context.trace_id
+            span_id = current_tracer.span_context.span_id
+            if not span_id:
+                span_id = span_context_module.generate_span_id()
+            span_context = span_context_module.SpanContext(
+                trace_id=trace_id, span_id=span_id
+            )
+            # Create and Saves Tracer and Sampler to ContextVar
+            sampler = samplers.ProbabilitySampler(
+                rate=self.config.sampling_rate
+            )
+            self.trace_exporter = trace_exporter.StackdriverExporter(
+                project_id=self.project_id,
+                transport=StackDriverAsyncTransport,
+            )
+            self.tracer = tracer.Tracer(
+                sampler=sampler,
+                span_context=span_context,
+                exporter=self.trace_exporter,
+            )
+
     def export_stats_data(
         self, stats_data: List[_observability.StatsData]
     ) -> None:
-        # TODO(xuanwn): Add implementation
-        raise NotImplementedError()
+        if not self.config.stats_enabled:
+            return
+        for data in stats_data:
+            measure = _views.METRICS_NAME_TO_MEASURE.get(data.name, None)
+            if not measure:
+                continue
+            # Create a measurement map for each metric, otherwise metrics will
+            # be override instead of accumulate.
+            measurement_map = self.stats_recorder.new_measurement_map()
+            # Add data label to default labels.
+            labels = data.labels
+            labels.update(self.default_labels)
+            tag_map = TagMap()
+            for key, value in labels.items():
+                tag_map.insert(TagKey(key), TagValue(value))
+
+            if data.measure_double:
+                measurement_map.measure_float_put(measure, data.value_float)
+            else:
+                measurement_map.measure_int_put(measure, data.value_int)
+            measurement_map.record(tag_map)
 
     def export_tracing_data(
         self, tracing_data: List[_observability.TracingData]
     ) -> None:
-        # TODO(xuanwn): Add implementation
-        raise NotImplementedError()
+        if not self.config.tracing_enabled:
+            return
+        for span_data in tracing_data:
+            # Only traced data will be exported, thus TraceOptions=1.
+            span_context = span_context_module.SpanContext(
+                trace_id=span_data.trace_id,
+                span_id=span_data.span_id,
+                trace_options=trace_options_module.TraceOptions(1),
+            )
+            span_datas = _get_span_datas(
```

Comment:
Nit: "s/datas/data/g" here and throughout. Data is either a mass noun (e.g. "some data") or (less commonly) the plural form of a count noun (e.g "this datum, these data")